Jenkinsfile

```diff
@@ -15,8 +15,12 @@ pipeline {
     timeout(time: 30, unit: 'MINUTES')
   }
   environment {
-    /** See https://issues.jenkins-ci.org/browse/JENKINS-42771 - we'd like to expand this out into multi-line concatenations */
-    PYTEST_ADDOPTS = "-n=10 --tb=short --color=yes --driver=SauceLabs --variables=capabilities.json"
+    PYTEST_ADDOPTS =
+    "-n=10 " +
```

Comment:
Nit: I think this looks better when it's indented two spaces. Not going to block on merging, though.